youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -420,17 +421,33 @@ def add(meta_list, info_list=None):
 
         filename = info['filepath']
         temp_filename = prepend_extension(filename, 'temp')
+        metadata_filename = replace_extension(filename, 'meta')
+
+        options = ['-map_metadata', '1']
 
         if info['ext'] == 'm4a':
-            options = ['-vn', '-acodec', 'copy']
+            options.extend(['-vn', '-acodec', 'copy'])
         else:
-            options = ['-c', 'copy']
+            options.extend(['-c', 'copy'])
 
+        metadata_file_content = ';FFMETADATA1\n'
         for (name, value) in metadata.items():
-            options.extend(['-metadata', '%s=%s' % (name, value)])
+            metadata_file_content += '%s=%s\n' % (name, value)
```

Comment:
As of [1]
>Metadata keys or values containing special characters (‘=’, ‘;’, ‘#’, ‘\’ and a newline) must be escaped with a backslash ‘\’.

Make sure multiline and exotic meta values are embedded correctly.

1. http://www.ffmpeg.org/ffmpeg-formats.html#Metadata-1